docs/changes.txt

```diff
@@ -1,4 +1,4 @@
--------------------------------------------------------------------------------
+﻿-------------------------------------------------------------------------------
```

Comment:
Not sure if this change was intentional?

Reply:
I added
3.0.5: (released ????-??-??)
----------------------------

wxMSW:

- Fix libtiff compilation with MinGW-w32 6.3.


before the 3.0.4: line

I don't see the above.

Reply:
Looks like you changed the encoding of the file (e.g. added an UTF byte order mask).

Reply:
Not sure how that would have happened. I have redone a fresh copy using linux to make the change instead of windows.